react-app/src/components/property/PropertyDetail.tsx

```diff
@@ -96,29 +114,45 @@ const PropertyDetail = (props: IPropertyDetail) => {
           title={val.Name}
         />
       );
-    } else if (key === 'IsSensitive') {
+    } else if (key === 'IsSensitive' || key === 'Owned') {
       return val ? <Typography>Yes</Typography> : <Typography>No</Typography>;
     }
-    return <></>;
+    return <Typography>{val}</Typography>;
   };
 
-  const buildingOrParcel = building ? 'Building' : 'Parcel';
+  const buildingOrParcel: PropertyType = typeof building === 'object' ? 'Building' : 'Parcel';
```

Comment:
It shouldn't happen because this page will be under a different route depending on whether it's a building or a parcel. 

`typeof` bit is just there because the fetch for building/parcel by id returns string if it's not found.

Reply:
Then I wonder if we can look at the url to see what path to call to find a building or parcel. Right now it always calls both.

Reply:
I've improved this in recent commits.